02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -32,12 +32,13 @@ Microarrays measure gene expression using chips filled with oligonucleotide prob
 After hybridization, the microarrays are scanned, and the fluorescence intensity for each probe is measured.
 The fluorescence intensity indicates the number of labeled fragments bound and therefore the relative quantity of the transcript the probe is designed for. 
 
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg" width=400>  
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/72ba39735440ea40051a3e753a71ae80f1094eac/components/figures/microarray-overview.png" width=400>
```

Comment:
```suggestion
<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/72ba39735440ea40051a3e753a71ae80f1094eac/components/figures/microarray-overview.png" width=600>
```

Now that they are pretty, we should make them bigger!! SO people can see the pretty work!